diff -up libdigidocpp-4.1.0/src/util/memory.h.omv~ libdigidocpp-4.1.0/src/util/memory.h
--- libdigidocpp-4.1.0/src/util/memory.h.omv~	2025-03-09 19:29:47.632688982 +0100
+++ libdigidocpp-4.1.0/src/util/memory.h	2025-03-09 19:29:53.710008423 +0100
@@ -33,7 +33,7 @@ constexpr unique_free_t<T> make_unique_p
 
 template<class T>
 [[nodiscard]]
-constexpr auto make_unique_ptr(nullptr_t, void (*d)(T*)) noexcept
+constexpr auto make_unique_ptr(std::nullptr_t, void (*d)(T*)) noexcept
 {
     return make_unique_ptr<T, T>(nullptr, d);
 }
